src/index.html

```diff
@@ -11,7 +11,142 @@
+        <label class="form__field">
+          Surname:
+          <input
+          type="text"
+          name="Surname"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form__field">
+          Name:
+          <input
+          type="text"
+          name="Name"
+          autocomplete="off"
+          >
+        </label>
+        <label class="form__field">
+          How old are you?
+          <input
+          type="number"
+          name="How old are You?"
+          value="12"
+          min="1"
+          max="100"
+          >
+        </label>
+        <label class="form__field">
+          Full date of birth:
+          <input
+          type="date"
+          name="Full date of birth"
+          >
+        </label>
+        <label class="form__field">
+          I accept the terms of the agreement:
+          <input type="checkbox" name="terms" >
+        </label>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Registration</legend>
+        <label class="form__field">
+          E-mail:
+          <input
+          type="email"
+          name="E-mail"
+          placeholder="email@example.com"
+          required
+          >
+        </label>
+        <label class="form__field">
+          Password:
+          <input
+          type="password"
+          name="Password"
+          minlength="6"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form__field">
+          Do you love cats?
+          <label>
+            <input
+            type="radio"
+            name="cats"
+            value="y"
+            >
+            YES
+          </label>
+          <label>
+            <input
+            type="radio"
+            name="cats"
+            value="n"
+            >
+            NO
+          </label>
+        </div>
+
+        <label class="form__field">
+          What is your favorite color?
+          <input type="color" name="color" >
+        </label>
+
+        <label class="form__field">
+          What time do you go to bed?
+          <input type="time" name="time_bed" >
+        </label>
+
+        <label class="form__field">
+          What are your favorite brand of cars?
+          <select name="fav_car" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+
+        <label class="form__field">
+          How do you rate our work?
+          <input
+          type="range"
+          name="rate_work"
+          value="0"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="group">
+        <legend>Additional info</legend>
+        <label class="form__field">
+          Comments:
+          <textarea
+          name="Comments"
+          maxlength="250">
+          </textarea>
```

Comment:
You don't need blank characters between closing and opening tags of textarea
```suggestion
          <textarea
            name="comments"
            maxlength="250"
         ></textarea>
```